lib/operations/collection_ops.js

```diff
@@ -846,7 +846,7 @@ function insertOne(coll, doc, options, callback) {
 function isCapped(coll, options, callback) {
   optionsOp(coll, options, (err, document) => {
     if (err) return handleCallback(callback, err);
-    handleCallback(callback, null, document && document.capped);
+    handleCallback(callback, null, document && !!document.capped);
```

Comment:
in case `document` is `undefined`, maybe we should be safe and make it `!!(document && document.capped)` ?

Reply:
imho this is fine: 
```
mbroadst@gorgor:~$ node
> !!undefined
false
> !!false
false
> !!true
true
```

Reply:
I meant more like

```js
const a = null;
const b = a && !!a.b;
console.log(b); // logs undefined;
```

Reply:
Looks like changing it to `!!(document && document.capped)` fixes the error in Mongo <= 2.6, where `document` is indeed `null` instead of an empty object

Reply:
nice